docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/1.insert-vertex.md

```diff
@@ -0,0 +1,79 @@
+# INSERT VERTEX
+
+```ngql
+INSERT VERTEX <tag_name> (<prop_name_list>) [, <tag_name> (<prop_name_list>), ...]
+     {VALUES | VALUE} VID: (<prop_value_list>[, <prop_value_list>])
+
+prop_name_list:
+  [prop_name [, prop_name] ...]
+
+prop_value_list:
+  [prop_value [, prop_value] ...]
+```
+
+The `INSERT VERTEX` statement inserts a vertex or vertices into a graph space in a Nebula Graph instance.
+
+* `tag_name` denotes the `tag` (vertex type), which must be created before `INSERT VERTEX`.
+* `prop_name_list` is the property name list in the given `tag_name`.
```

Comment:
This may be better understood:
`prop_name_list` contains the names of the properties attached to the tag.